Getting Asynchronous section has invalid example

The second example here:

* https://tokio.rs/docs/futures/getting_asynchronous/

If fails to compile with:

```
error[E0271]: type mismatch resolving `<hello::HelloWorld as futures::Future>::Error == ()`
  --> src/hello/mod.rs:43:5
   |
43 |     tokio::run(hello_world)
   |     ^^^^^^^^^^ expected struct `std::io::Error`, found ()
   |
   = note: expected type `std::io::Error`
              found type `()`
   = note: required by `tokio::run`

error: aborting due to 2 previous errors

Some errors occurred: E0271, E0432.
For more information about an error, try `rustc --explain E0271`.
```

Error cannot be non-empty tuple for `tokio::run()`, plus `poll` does not refer to `Self::Error`.

--- a/content/docs/futures/getting_asynchronous.md
+++ b/content/docs/futures/getting_asynchronous.md
@@ -134,10 +134,10 @@ fn main() {
     let addr = "127.0.0.1:1234".parse().unwrap();
     let connect_future = TcpStream::connect(&addr);
     let hello_world = HelloWorld::Connecting(connect_future);
-# let hello_world = futures::future::ok::<(), ()>(());
+# let hello_world = futures::future::ok::<(), io::Error>(());
 
-    // Run it
-    tokio::run(hello_world)
+    // Run it, here we map the error since tokio::run expects a Future<Item=(), Error=()>
+    tokio::run(hello_world.map_err(|e| println!("{0}", e)))
 }
 ```
 

